Modelica/Magnetic/FluxTubes/Examples/BasicExamples/ToroidalCoreAirgap.mo

```diff
@@ -0,0 +1,118 @@
+    annotation (Line(points={{50,-10},{60,-10}}, color={0,0,255}));
+  connect(excitingCoil.port_n, magFluxSensor.port_n)
+    annotation (Line(points={{-20,-10},{-20,-20}}, color={255,127,0}));
+  connect(magFluxSensor.port_p, magneticGround.port)
+    annotation (Line(points={{-20,-40},{-20,-50}}, color={255,127,0}));
+  connect(magneticGround.port, airGap.port_n)
+    annotation (Line(points={{-20,-50},{-10,-50}}, color={255,127,0}));
+  connect(excitingCoil.port_p, core.port_p)
+    annotation (Line(points={{-20,10},{-20,30},{-10,30}}, color={255,127,0}));
+  annotation (Documentation(info="<html>
+<p>
+Educational example of a magnetic circuit containing a toroidal iron core with circular cross section and an airgap:
+</p>
+<p>
+A current ramp is applied in positive electric direction through the exciting coil, causing a rising magnetomotive force (mmf) in positive magnetic direction of the electromagnetic converter.  
+The mmf in turn causes a magnetic flux through the circuit in the direction indicated by the flux sensor. 
+From that magnetic flux, flux density can be calculated in every element of the magnetic circuit. Flux density is used to derive magnetic field strength.
+Magnetic field strength times length of the flux line gives magnetic potential difference of each element. 
+The sum of all magnetic potential differences is covered by the mmf of the exciting coil.
+</p>
+<p>
+Using the values shown in section Parameters, the results can be validated easily by analytic calculations:
+</p>
+<table>
+<tr><th>element   </th><th>cross section     </th><th>length             </th><th>material          </th><th>B                   </th><th>H                                    </th><th>mmf              </th></tr>
+<tr><td>core      </td><td>d<sup>2</sup>*pi/4</td><td>r*alfa             </td><td>&mu;<sub>r</sub>  </td><td>flux / cross section</td><td>B/(&mu;<sub>r</sub>*&mu;<sub>0</sub>)</td><td>H*length         </td></tr>
+<tr><td>airgap    </td><td>d<sup>2</sup>*pi/4</td><td>delta=r*(2*pi-alfa)</td><td>&mu;<sub>r</sub>=1</td><td>flux / cross section</td><td>B/(&mu;<sub>r</sub>*&mu;<sub>0</sub>)</td><td>H*length         </td></tr>
+<tr><td>total     </td><td>                  </td><td>                   </td><td>                  </td><td>                    </td><td>                                     </td><td>&Sigma; mmf = N*I</td></tr>
+</table>
+<p>
+Note that since no leakage is present, the magnetic flux is the same in every element - they are connected in series. 
+For calculation of the length of flux lines, a medium flux line is used.
+</p>
+<p>
+Additonally, a measuring coil is placed in the airgap. 
+Due to Faraday's law, the time derivative of flux causes an induced voltage both in the exciting coil (in positive direction) and in the measuring coil (in negative direction). 
+Since current and therefore flux are a linear time dependent ramp, induced voltages are constant during that ramp and zero otherwise. 
```

Comment:
Proposal: .. is given as a linear time-dependent ramp, the induced voltages during this ramp are constant and otherwise zero.

Reply:
resolved by 31b0ebc2c..4772a8ba6